planetarium/NineChronicles
# Apply level limit data on crafting.

### Description

SSIA

### How to test

1. Go to `Assets/Resources/AddressableAssetsContainer.asset` and add `ItemRequirementSheet.csv` to `TableCsvAssets`.
2. Check if level limit text is properly displayed in `UI_Craft` and `UI_UpgradeEquipment`.

### Related Links
https://app.asana.com/0/958521740385861/1201705327357531

### Required Reviewers
@planetarium/9c-dev @BBeom06 

### Screenshot
![image](https://user-images.githubusercontent.com/26648527/152534256-f4fdd76a-a48f-4b7a-960d-390c4c29e3eb.png)
![image](https://user-images.githubusercontent.com/26648527/152534305-5ba19798-6f85-432b-818f-9ba4766b2585.png)
